recipes/ferret_datasets/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "7.0.0" %}
+
+package:
+    name: ferret_datasets
+    version: {{ version }}
+
+source:
+    fn: ferret_datasets-{{ version }}.tar.gz
+    url: https://github.com/NOAA-PMEL/PyFerret/releases/download/v{{ version }}-final/FerretDatasets.tar.gz
+    sha256: 3d735fd3e952e01646894beea91b39c2adbfde087bcfec2a78ebef189048213f
+
+build:
+    number: 0
+    skip: True  # [win or py3k or osx]
+
+requirements:
+    build:
+        - python
+        - pyferret
+    run:
+        - python
+        - pyferret
+
+test:
+    imports:
+        - pyferret
+
+about:
+    home: http://ferret.pmel.noaa.gov/Ferret
+    license: MIT
```

Comment:
@karlmsmith I used the same license as `pyferret`, but ideally we should list all the data licenses (if any) here.